Library/Homebrew/utils.rb

```diff
@@ -388,6 +388,10 @@ def exec_browser(*args)
   browser ||= OS::PATH_OPEN if defined?(OS::PATH_OPEN)
   return unless browser
 
+  if ENV["HOMEBREW_DISPLAY"]
```

Comment:
Can skip the `if`; won't do any harm if it's not set.

Reply:
My personal preference would be…
```ruby
ENV["DISPLAY"] = ENV["HOMEBREW_DISPLAY"] if ENV["HOMEBREW_DISPLAY"]
```
The error message from a browser may be different if the `DISPLAY` environment variable is not set versus set to the empty string.
@mtmiller Could you please test…
```sh
env -u DISPLAY firefox
env DISPLAY= firefox
```
and see what the error message is?

Reply:
Thanks for the feedback. I eliminated the `if` entirely. I'm not much of a rubyist, but I found that `ENV` DTRT, assigning `nil` deletes from the environment.

Reply:
> Could you please test…
> and see what the error message is?

I did, the error message does differ by browser, but the effect is the same, no tab opened.

Firefox does differentiate:

```
$ env -u DISPLAY firefox
Error: no DISPLAY environment variable specified
$ env DISPLAY= firefox
Unable to init server: Broadway display type not supported: 
Error: cannot open display: 
```

Chrome doesn't care whether it's unset or empty:

```
$ env -u DISPLAY google-chrome

(google-chrome:32416): Gtk-WARNING **: 12:48:05.569: cannot open display: 
$ env DISPLAY= google-chrome

(google-chrome:32480): Gtk-WARNING **: 12:48:09.083: cannot open display: 
```